spring/src/test/java/org/axonframework/spring/eventhandling/deadletter/SpringJpaDeadLetteringIntegrationTest.java

```diff
@@ -16,19 +16,19 @@
 
 package org.axonframework.spring.eventhandling.deadletter;
 
-import org.axonframework.common.jpa.EntityManagerProvider;
-import org.axonframework.common.jpa.SimpleEntityManagerProvider;
+import org.axonframework.common.legacyjpa.EntityManagerProvider;
```

Comment:
I assume there's no equivalent of this in the `hibernate-6-integrationtests` module, as that module doesn't include Spring, right?

Reply:
Yes, but added one in `axon-spring-boot-3-integration-tests`.